### DISCUSSION TITLE
MinIO Service unable to start

### DISCUSSION THREAD

**Discussion by SriRamJai8** [minio/minio#20177] [Q&A]
Hi 

Do we have any option to graceful shutdown of MinIO before I reboot the OS. 

I am running into below error, after rebooting OS I use " minio server /mnt/disk1 /mnt/disk2 /mnt/disk3 --console 9001 to start teh minio service after restart. 

If I remove the .minio.sys file, I will be able to start the service, However my Users, Secretkey etc will be lost. I have to recreate them.

![image](https://github.com/user-attachments/assets/ebcb9001-f0af-4298-92aa-8d5b2573d2c4)

**Comment by harshavardhana** [comment]
are you using NFS?

**Reply by SriRamJai8** [reply]
Hi Harsh.. 

Thank you for the quick response, Yes I am using the network share. does the system rename the minio.sys file during every startup ?

**Reply by harshavardhana** [reply]
Correct this issue because of NFS. You need to use 

- Atleast NFS `vers=4` 
- Or turn client side caching with NFSv3

**Reply by SriRamJai8** [reply]
Thanks again, I will test the recommendation, and keep you posted, Meanwhile could you please help me on the tiering QA. 

I have added multiple targets to tiering, how lifecycle policy will handle the transition to multiple ILM targets
